modules/fb_instant_articles_rss/fb_instant_articles_rss.module

```diff
@@ -178,49 +178,42 @@ function fb_instant_articles_rss_form_node_form_alter(&$form, &$form_state, $for
     }
 
     // Is an Article?
-    $type = $form['#node']->type;
-    if (!empty($form['#node']->nid)) {
-      $fb_instant_enabled = fb_instant_articles_rss_is_article($form['#node']->nid);
-    }
-    else {
-      $fb_instant_enabled = fb_instant_articles_display_is_article_type('node', $type);
-    }
+    $is_type = fb_instant_articles_display_is_article_type('node', $form['#node']->type);
```

Comment:
This check is already made in the outer `if` statement, so you can assume it's `TRUE`, and don't need to do it again.